mpto/src/parser_2018.py

```diff
@@ -0,0 +1,167 @@
+import pandas as pd
+from datetime import datetime
+import math
+import pathlib
+import sys
+import os
+import utils
+
+# Read data downloaded from the crawler
+
+def read_data(path):
+    try:
+        data = pd.read_excel(path, engine=None)
+        return data
+    except Exception as excep:
+        sys.stderr.write(
+            "'Não foi possível ler o arquivo: "
+            + path
+            + ". O seguinte erro foi gerado: "
+            + excep
+        )
+        os._exit(1)
+
+
+# Strange way to check nan. Only I managed to make work
+# Source: https://stackoverflow.com/a/944712/5822594
+
+def isNaN(string):
```

Comment:
Mesma sugestão acima para `is.Nan`

Reply:
Por `is` ser palavra reservada em python, escolhi manter a função `is_nan` no arquivo `utils.py`. Tudo bem assim?